assets/src/application/ui/display/Popover/style.scss

```diff
@@ -0,0 +1,14 @@
+.ee-popover {
+	&__content {
+		padding: var(--ee-padding-small);
+
+		&:focus {
+			outline: 0;
+			box-shadow: 0 0 0 3px rgba(66, 153, 225, 0.6);
+		}
+
+		.chakra-popover__arrow {
+			display: none;
```

Comment:
these styles were needed because this is how it was looking when popover was imported from `@chakra-ui/popover`:

![image](https://user-images.githubusercontent.com/1477453/85619977-c6b92600-b66b-11ea-8b95-118bce75c064.png)

Reply:
here is also seen how arrow is positioned wrongly